src/sqlfe/util/Utilities.java

```diff
@@ -226,4 +228,32 @@ public static String processSlashes(String path) {
 		return resultPath;
 	}	// end - method processSlashes
 	
+	public static boolean isQuestionFound(String line){
+		String regexp = "-- -- \\d+[a-z]*[.]";		// regular expression for question, e.g. >-- --1a. or -- --23.<
```

Comment:
There is an extra slash here that is messing up the regex, it's looking for a literal slash character.
```suggestion
		String regexp = "-- -- \d+[a-z]*[.]";		// regular expression for question, e.g. >-- --1a. or -- --23.<
```
Also, should we make the period optional? Might be a question for Paul

Reply:
So, there are two issues with the changes you suggested
1. I thought about it but having a carat(^) would make the pattern a little inflexible. For ex: the pattern would reject any lines which have a space before the instructor comment "-- --". Than we would have to add a line.trim() before trying to match, extra operation that is avoided now.
2. Having an extra slash is necessary in Java regex because single slash is translated as an escape sequence. So the line 
String regexp = "^-- -- \d+[a-z]*[.]";	
throws an error as the escape sequence \d+[a-z]*[.] is invalid .

Reply:
ah gotcha, thanks for clarifying! I'm fine with leaving this as is then